components/sendgrid/actions/get-all-bounces/get-all-bounces.js

```diff
@@ -0,0 +1,59 @@
+const validate = require("validate.js");
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-get-all-bounces",
+  name: "Get All Bounces",
+  description: "Allows you to get all of your bounces.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    startTime: {
+      type: "integer",
+      label: "Start Time",
+      description:
+        "Refers start of the time range in unix timestamp when a bounce was created (inclusive).",
+      optional: true,
+    },
+    endTime: {
+      type: "integer",
+      label: "End Time",
+      description:
+        "Refers end of the time range in unix timestamp when a bounce was created (inclusive).",
+      optional: true,
+    },
+  },
+  methods: {
+    ...common.methods,
+  },
+  async run() {
```

Comment:
Include the `$` variable.
```suggestion
  async run({ $ }) {
```